tests/integration/any_charm.py

```diff
@@ -27,21 +27,19 @@ def __init__(self, *args, **kwargs):
             kwargs: Variable list of positional keyword arguments passed to the parent constructor.
         """
         super().__init__(*args, **kwargs)
-        self.ingress = IngressRequires(
-            self,
-            {"service-hostname": self.app.name, "service-name": self.app.name, "service-port": 80},
-        )
+        require_nginx_route(charm=self, service_hostname=self.app.name, service_name=self.app.name, service_port=80)
 
-    def update_ingress(self, ingress_config):
-        """Update Ingress config.
+    def delete_nginx_route_relation_data(self, field: str) -> None:
+        """Delete one data filed from the nginx-route relation data.
 
         Args:
-            ingress_config: New Ingress configuration to be applied.
+            field: the name of the field to be deleted.
         """
-        self.ingress.update_config(ingress_config)
+        relation = self.model.get_relation("nginx-route")
+        del relation.data[self.app][field]
 
     @staticmethod
-    def start_server(port: int = 80):
+    def start_server(port: int = 8080):
```

Comment:
Why change this?

Reply:
That shouldn't be changed, I copy-pasted another any-charm file and forgot to change the port, thanks for spotting that!